/*------------------------------------------------------------------------*
 * Copyright (c) 2026 Sonu Sonkar.                                        *
 * Licensed under the MIT License.                                        *
 * See the LICENSE file in the project root for full license information. *
 *------------------------------------------------------------------------*/
syntax = "proto3";

service CrisisService {
  rpc Evaluate (CrisisRequest) returns (CrisisResponse);
}

message CrisisRequest {
  repeated float kpi_sequence = 1;  // flattened (1, T, F)
  int32 seq_len = 2;                // T
  int32 feature_dim = 3;            // F
}

message CrisisResponse {
  repeated float crisis_scores = 1; // length = T
}
